Polyfills/Window/Source/Windows/WindowPlatform.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <Windows.h>
+#include <Babylon/JsRuntime.h>
+
+#include "..\Window.h"
```

Comment:
You should be able to do this just as `<Window.h>`; CMake should have set up the include directories so that you can reference header without having to depend directly on the directory relationship to it.

Reply:
Yes, that's definitely a better suggestion.